src/internet-analyzer/azext_internet_analyzer/vendored_sdks/frontdoor/models/_front_door_management_client_enums.py

```diff
@@ -0,0 +1,318 @@
+    https_only = "HttpsOnly"
+    match_request = "MatchRequest"
+
+
+class BackendEnabledState(str, Enum):
+
+    enabled = "Enabled"
+    disabled = "Disabled"
+
+
+class FrontDoorHealthProbeMethod(str, Enum):
+
+    get = "GET"
+    head = "HEAD"
+
+
+class HealthProbeEnabled(str, Enum):
+
+    enabled = "Enabled"
+    disabled = "Disabled"
+
+
+class SessionAffinityEnabledState(str, Enum):
+
+    enabled = "Enabled"
+    disabled = "Disabled"
+
+
+class ResourceType(str, Enum):
+
+    microsoft_networkfront_doors = "Microsoft.Network/frontDoors"
+    microsoft_networkfront_doorsfrontend_endpoints = "Microsoft.Network/frontDoors/frontendEndpoints"
+
+
+class Availability(str, Enum):
+
+    available = "Available"
+    unavailable = "Unavailable"
+
+
+class PolicyEnabledState(str, Enum):
+
+    disabled = "Disabled"
+    enabled = "Enabled"
+
+
+class PolicyMode(str, Enum):
+
+    prevention = "Prevention"
+    detection = "Detection"
+
+
+class CustomRuleEnabledState(str, Enum):
+
+    disabled = "Disabled"
+    enabled = "Enabled"
+
+
+class RuleType(str, Enum):
+
+    match_rule = "MatchRule"
+    rate_limit_rule = "RateLimitRule"
+
+
+class MatchVariable(str, Enum):
+
+    remote_addr = "RemoteAddr"
+    request_method = "RequestMethod"
+    query_string = "QueryString"
+    post_args = "PostArgs"
+    request_uri = "RequestUri"
+    request_header = "RequestHeader"
+    request_body = "RequestBody"
+    cookies = "Cookies"
+    socket_addr = "SocketAddr"
+
+
+class Operator(str, Enum):
+
+    any = "Any"
+    ip_match = "IPMatch"
+    geo_match = "GeoMatch"
+    equal = "Equal"
+    contains = "Contains"
+    less_than = "LessThan"
+    greater_than = "GreaterThan"
+    less_than_or_equal = "LessThanOrEqual"
+    greater_than_or_equal = "GreaterThanOrEqual"
+    begins_with = "BeginsWith"
+    ends_with = "EndsWith"
+    reg_ex = "RegEx"
+
+
+class TransformType(str, Enum):
+
+    lowercase = "Lowercase"
+    uppercase = "Uppercase"
+    trim = "Trim"
+    url_decode = "UrlDecode"
+    url_encode = "UrlEncode"
+    remove_nulls = "RemoveNulls"
+
+
+class ActionType(str, Enum):
+
+    allow = "Allow"
+    block = "Block"
+    log = "Log"
+    redirect = "Redirect"
+
+
+class ManagedRuleExclusionMatchVariable(str, Enum):
+
+    request_header_names = "RequestHeaderNames"
+    request_cookie_names = "RequestCookieNames"
+    query_string_arg_names = "QueryStringArgNames"
+    request_body_post_arg_names = "RequestBodyPostArgNames"
+
+
+class ManagedRuleExclusionSelectorMatchOperator(str, Enum):
+
+    equals = "Equals"
+    contains = "Contains"
+    starts_with = "StartsWith"
+    ends_with = "EndsWith"
+    equals_any = "EqualsAny"
+
+
+class ManagedRuleEnabledState(str, Enum):
+
+    disabled = "Disabled"
+    enabled = "Enabled"
+
+
+class PolicyResourceState(str, Enum):
+
+    creating = "Creating"
+    enabling = "Enabling"
+    enabled = "Enabled"
+    disabling = "Disabling"
+    disabled = "Disabled"
+    deleting = "Deleting"
```

Comment:
I don't think that we need any of these

Reply:
well, this is a part of python SDK package, so I think I won't be changing that